gradio/blocks.py

```diff
@@ -519,8 +519,8 @@ def __init__(
         self,
         theme: str = "default",
         analytics_enabled: bool | None = None,
-        mode: str = "blocks",
-        title: str = "Gradio",
+        mode: str | None = "blocks",
```

Comment:
why allow `blocks` and `mode` to be null? Not clear what null even means in this case.

Reply:
Sounds good, will change